server/src/main/java/org/opensearch/identity/tokens/AuthToken.java

```diff
@@ -14,5 +14,5 @@
  * @opensearch.experimental
  */
 public interface AuthToken {
-
+    String getTokenValue();
```

Comment:
Seems like this is suppose to be a value that can be provided in the http header called authentication, maybe rename to asAuthenticationHeaderValue() to make it clear how what is expected?